backend/src/main/scala/bloop/Compiler.scala

```diff
@@ -874,8 +874,17 @@ object Compiler {
       case _ => false
     }
     val updatedClasspath =
-      if (needsRtJar) inputs.classpath ++ RtJarCache.create(JavaRuntime.version, logger)
-      else inputs.classpath
+      if (needsRtJar) {
+        val possibleRtJar = inputs.javacBin
+          .map { binary =>
+            binary.getParent.resolve("../jre/lib/rt.jar")
```

Comment:
It looks suspicious to do `getParent` and then resolve `..`. Would it be better to do `getParent.getParent`, or `../..`?

Reply:
Switched to getParent and wrapped in Try (if someone has java bin in `/java`, which sounds ridiculous, but I've seen things.)